### ISSUE TITLE
add buttons for default todo tasks

### ISSUE THREAD

**Issue by valerievozza** [AdamRobinsonSE/group-project#8]
Add buttons for tasks such as
* Food
* Water
* Medicine
* Play
* Scoop litter
* Change litter

Buttons should have image and text for accessibility. Alt text may be sufficient

**Comment by raissa-k** [comment]
I'll take this one!
